tree-sitter-darklang/grammar.js

```diff
@@ -784,6 +795,156 @@ module.exports = grammar({
         field("body", $.expression),
       ),
 
+    // ---------------------
+    // Pipe expressions
+    // ---------------------
+
+    pipe_variable_or_fn_call: $ =>
+      seq(field("variable_or_fn_name", $.variable_or_fn_identifier)),
+
+    pipe_lambda: $ =>
```

Comment:
should `$.lambda_expression` be reused here?
i.e.
```
pipe_lambda: $ =>
      choice(
        $.lambda_expression,
        seq(
          field("symbol_open_paren", alias("(", $.symbol)),
          $.lambda_expression,
          field("symbol_close_paren", alias(")", $.symbol)),
        ),
      ),
      ```

Reply:
Yeah, I think so. I assume the same thing is reused in both equivalent parts of PT? if so, then yes.